File: system/src/main/resources/static/system/js/service/shopService.js
app.service("shopService", function ($http) {
    // 条件查询
    this.search = function (page, rows, searchEntity) {
        return $http.post(basePath + "/shop/apply/list?page=" + page + "&rows=" + rows,
            searchEntity);
    };
    this.apply = function (ids,status) {
        return $http.get(basePath + "/shop/examine?ids="+ids +"&status=" + status);
    };
});

File: system/src/main/java/com/cys/system/common/service/ShopService.java
package com.cys.system.common.service;

import com.aliyuncs.exceptions.ClientException;
import com.cys.system.common.common.pojo.Result;

public interface ShopService {
    Result listApplyShop(Integer page, Integer rows);

    Result examine(Integer[] ids, Integer status);
}
